src/backend/env/post.rs

```diff
@@ -621,52 +618,33 @@ impl Post {
         Ok(id)
     }
 
-    pub fn count(state: &State) -> usize {
-        state.posts.len() + state.memory.posts.len()
+    pub fn count() -> u64 {
+        POSTS.with(|p| p.borrow().as_ref().unwrap().len())
     }
 
-    // Get the post from the heap if available, or load from the stable memory into the cache and
-    // return the reference to it
-    pub fn get<'a>(state: &'a State, post_id: &PostId) -> Option<&'a Post> {
-        state.posts.get(post_id).or_else(|| {
-            let boxed = cache().get(post_id).or_else(|| {
-                state.memory.posts.get(post_id).and_then(|mut post: Post| {
-                    let cache = cache();
-                    post.archived = true;
-                    cache.insert(*post_id, Box::new(post));
-                    cache.get(post_id)
-                })
-            });
-            boxed.map(|ptr| &**ptr)
-        })
+    pub fn get(post_id: &PostId) -> Option<Post> {
+        POSTS.with(|p| p.borrow().as_ref().unwrap().get(post_id))
```

Comment:
That's correct. But the migration is expected to last only a couple of minutes. Is it worth adding complexity to bridge this downtime?

Reply:
It is a couple of lines of code that would keep the webpage functioning in read-only mode: if the post is in state.posts then return it otherwise use the current code?